### ISSUE TITLE
Arducopter on WeAct Studio STM32F405RGT6 with GY-87 only:  "config error : unable to initialize driver" and QuadX not recognized

### ISSUE THREAD

**Issue by jct1411** [rizacelik/STM32F405RGT6_INAV_Firmware#15]
Dear Sir,

I setup some config with WeAct Studio STM32F405RGT6 Board on a breadboard with GY-87 only.  I flashed the board with the provided Arducopter firmware.  Connection  with latest stable version of Mission Planner is working but  I get " config error : unable to initialize driver" and going into the setup QuadX is not recognized.  What could be the problem ?

![Arducopter-1](https://github.com/user-attachments/assets/bff963ed-3107-4961-97fb-152368f6cb50)
![Arducopter-2](https://github.com/user-attachments/assets/447734d4-2172-4663-97ff-0e23d66b5a66)

Thanks for your help

**Comment by rizacelik** [comment]
Use a sensor other than GY-87. GY-87 also works with MPU6050 I2C and is quite slow. This sensor is no longer supported.

**Comment by jct1411** [comment]
I presume a new build of the Arducopter firmware is required which supports GY-91 ?  How do I connect GY-91 ?  Same cabling as for Betaflight ?  But you mention that this the circuit connections is for betaflight

Thanks

**Comment by jct1411** [comment]
I connected GY-91 the same as for Betaflight.  When connected, Mission Planner 1.3.82 reports now explicitely "Config Error : Baro: unable to initialize driver" and "Config Error: fix problem then reboot".  
QuadX is still not recognized.

**Comment by rizacelik** [comment]
There is no GY-91 support for Arducopter at the moment. If I find time, I will add support. Unfortunately, I don't have time.

**Comment by shrinibas7** [comment]
> Use a sensor other than GY-87. GY-87 also works with MPU6050 I2C and is quite slow. This sensor is no longer supported.

Hey riza ,i added bmi270 works fine in inav but saying config error ins in ardupilot pls help…

**Comment by synersignart** [comment]
![Image](https://github.com/user-attachments/assets/96875e4d-ac62-4e3c-9a67-6e74b059c815)
imusing the BMI160 HMC5883 and BMP280
also getting configure error

**Comment by shrinibas7** [comment]
> ![Image](https://github.com/user-attachments/assets/96875e4d-ac62-4e3c-9a67-6e74b059c815) imusing the BMI160 HMC5883 and BMP280 also getting configure error

Hey synerderg🇵🇭 same thing happened with me bmi270 worked with inav but not with ardupilot…

**Comment by synersignart** [comment]
> > ![Image](https://github.com/user-attachments/assets/96875e4d-ac62-4e3c-9a67-6e74b059c815) imusing the BMI160 HMC5883 and BMP280 also getting configure error
> 
> Hey synerderg🇵🇭 same thing happened with me bmi270 worked with inav but not with ardupilot…

yup  strange because I also have my IMU on ISP2 and the sensors Riza listed are also supported
